tensorflow/buildspec.yml

```diff
@@ -3,7 +3,7 @@
   framework: &FRAMEWORK tensorflow
   version: &VERSION 2.3.1
   short_version: &SHORT_VERSION 2.3
-  cuda_version: &CUDA_VERSION cu110
+  cuda_version: &CUDA_VERSION cu102
```

Comment:
Why revert this?

Reply:
wanted to ensure this worked for cu102, put back to cu110